src/Execution/MutationExecutor.php

```diff
@@ -107,26 +187,22 @@ public static function executeUpdate(Model $model, Collection $args, HasMany $pa
 
         $model = self::saveModelWithBelongsTo($model, $remaining, $parentRelation);
 
-        $hasMany->each(function ($nestedOperations, string $relationName) use ($model) {
+        $hasMany->each(function ($nestedOperations, $relationName) use ($model) {
             /** @var HasMany $relation */
             $relation = $model->{$relationName}();
 
-            collect($nestedOperations)->each(function ($values, string $operationKey) use ($relation) {
-                if ($operationKey === 'create') {
+            collect($nestedOperations)->each(function ($values, $operationKey) use ($relation) {
```

Comment:
Why did you remove the `string` type hint?

Reply:
Messed up branches, this is old commit